cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html

```diff
@@ -219,7 +219,7 @@
     'nav_items': [
         {'text': 'Consumer Credit Trends', 'url': '/data-research/consumer-credit-trends/'},
         {'text': 'Credit Card Surveys & Agreements', 'url': '/data-research/credit-card-data/'},
-        {'text': 'Financial Well-Being Survey', 'url': '/data-research/financial-well-being-survey'},
+        {'text': 'Financial Well-Being Survey', 'url': '/data-research/financial-well-being-survey-data'},
```

Comment:
Trivial: Please add closing slash for consistency and to save Django the trouble :)

Reply:
Done, thanks!